FBTweak/_FBTweakBindObserver.m

```diff
@@ -12,16 +12,18 @@
 #import "FBTweak.h"
 #import "_FBTweakBindObserver.h"
 
-@interface _FBTweakBindObserver () <FBTweakObserver>
+static NSString * const kFBTweakCurrentValueKeyPath = @"tweak.currentValue";
+
+@interface _FBTweakBindObserver ()
 @end
```

Comment:
`_FBTweakBindObserver ()` can be omitted.

Reply:
Done